sw/device/silicon_creator/lib/manifest.h

```diff
@@ -19,6 +19,18 @@
 extern "C" {
 #endif  // __cplusplus
 
+// Extern declarations for code_start, code_end, and entry_point in the
+// manifest.
+extern const uint32_t _text_start;
+extern const uint32_t _text_end;
```

Comment:
I think these should go inside the macro. Actually, I'd prefer removing the macro and moving this definition to a separate file, e.g. `manifest_def.c`, that should be used for in-flash boot stages (rom_ext and first owner boot stage the skeleton example we have is called `bare_metal`).

Reply:
Thanks for this input @alphan! Are you thinking something like this https://github.com/jon-flatley/opentitan/commit/1ef137eb9e5d377197c9df577cb0b7e233f43c1a? With something similar done for `bare_metal`.

Reply:
Yes, thanks @jon-flatley! But I'm not sure if we need the MANIFEST_DEF macro. Were you planning to use it in another place?